front_end/content.py

```diff
@@ -248,6 +261,95 @@ def get_problem_details(course, assignment, problem, format_content=False, parse
 
     return problem_dict
 
+def split_url(file_path):
```

Comment:
Please make this function name more descriptive. The function name is very generic, but it seems that it is accomplishing a specific purpose. Also consider whether content.py is the best place for this function.  If this function is specific to log summarization, it should go in that script. But if it can be used more generally, it can go here.